lib/FrontendTool/FrontendTool.cpp

```diff
@@ -330,6 +328,27 @@ class JSONFixitWriter
       AllEdits.push_back({SM, Fix.getRange(), Fix.getText()});
     }
   }
+
+  bool finishProcessing() override {
+    std::error_code EC;
+    std::unique_ptr<llvm::raw_fd_ostream> OS;
+    OS.reset(new llvm::raw_fd_ostream(FixitsOutputPath,
+                                      EC,
+                                      llvm::sys::fs::F_None));
+    if (EC) {
+      // Create a temporary diagnostics engine to print the error to stderr.
```

Comment:
I feel like there should be some shorter way to do this, since you have it twice. But that can happen later.